### ISSUE TITLE
[Question] Unable to submit VQAv2 result file to the evaluation server

### ISSUE THREAD

**Issue by XiaoruiMaLU** [haotian-liu/LLaVA#1607]
### Question

When I try to upload the vqav2 result file to the evaluation server https://eval.ai/web/challenges/challenge-page/830/my-submission, after I select the phase, the page will jump to the email verification page, which I have already done when I signed up, and I don't receive any new email after this page shows. I would like to know whether there is any solution to this or another way to evaluate the vqav2 result, thanks!
![截屏2024-07-16 01 18 44](https://github.com/user-attachments/assets/d3812437-0b4e-4ff5-b586-528510a105d1)

![截屏2024-07-16 01 18 51](https://github.com/user-attachments/assets/e68c7f6b-945b-48f3-96ab-70bdd6e10395)

**Comment by XiaoruiMaLU** [comment]
It shows https://eval.ai/web/permission-denied in the link.

**Comment by itsqyh** [comment]
I encountered the same problem with you, have you already addressed it?

**Comment by makanju0la** [comment]
To solve this, click on Participate and then click on Create New Team to create a team. You would to have a team account in order to submit your result.

**Comment by ShristiDasBiswas** [comment]
Hi, my submission is stuck on submitted for 2 days now. Has anybody encountered this issue recently? Thanks!

**Comment by jhjangjh** [comment]
> Hi, my submission is stuck on submitted for 2 days now. Has anybody encountered this issue recently? Thanks!

Me too

**Comment by ShristiDasBiswas** [comment]
Have you found any resolution venues/response from Eval AI team?

**Comment by jhjangjh** [comment]
> Have you found any resolution venues/response from Eval AI team?

No :(

**Comment by ShristiDasBiswas** [comment]
Hi, did your submission finally run? I had reached out to the eval AI team but they haven't responded yet.

**Comment by jhjangjh** [comment]
No, it’s still stuck. I’m just waiting at this point. Please let et me know if you get any updates from your side, and I’ll do the same for you!

**Comment by jhjangjh** [comment]
My submission were done!

**Comment by xuankunyang** [comment]
Hi, there! I'm stuck with the same problem. Is waiting for the evaluation results from EvalAI our only option? I've been waiting for almost 5 days for my evaluation results.

**Comment by jhjangjh** [comment]
Me too. It stops again

**Comment by ShristiDasBiswas** [comment]
Hi,


Please email visualqa@gmail.com which is the VQAv2 team with this issue. Maybe letting them know that this is a recurring problem might allow for alternative evaluation solutions besides EvalAI.


________________________________
